storagenode/storagenodedb/infodb.go

```diff
@@ -44,8 +44,8 @@ func newInfo(path string) (*InfoDB, error) {
 	dbutil.Configure(db, mon)
 
 	infoDb := &InfoDB{db: db}
-	infoDb.pieceinfo = pieceinfo{infoDb, spaceUsed{sync.Once{}, 0}}
-	infoDb.bandwidthdb = bandwidthdb{infoDb, bandwidthUsed{sync.RWMutex{}, time.Time{}, 0}}
+	infoDb.pieceinfo = pieceinfo{infoDb, spaceUsed{0, sync.Once{}}}
```

Comment:
okay for this PR buuuut


definitely a strong preference to not create structs with unnamed, positional fields. always always use named fields in struct creation.

Reply:
fixed